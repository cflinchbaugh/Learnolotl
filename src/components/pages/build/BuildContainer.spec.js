import React from 'react';
import {shallow} from 'enzyme';
import {BuildContainer} from './BuildContainer';
import ShallowRenderer from 'react-test-renderer/shallow';

describe('BuildContainer', () => {
    it('renders and matches snapshot', () => {
        const renderer = new ShallowRenderer();

        const buildContainerData = {

        }

        const tree = renderer.render(
            <BuildContainer {...buildContainerData}/>
        );

        expect(tree).toMatchSnapshot();
    });

    
});