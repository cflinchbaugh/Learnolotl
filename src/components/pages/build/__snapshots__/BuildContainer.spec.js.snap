// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BuildContainer renders and matches snapshot 1`] = `
<styled.div
  className="build-container-wrapper col-xs-12 clearfix mdCard"
>
  <BuildLandingView
    handleBuildLandingViewNext={[Function]}
  />
</styled.div>
`;
